tools/dataProtector/advanced/dps-smart-contract.md

```diff
@@ -9,4 +9,107 @@ data, and their owners.
 ## Code
 
 You can find the Solidity code here:
-https://github.com/iExecBlockchainComputing/dataprotector-sdk/tree/v2/packages/sharing-smart-contract
+<https://github.com/iExecBlockchainComputing/dataprotector-sdk/tree/v2/packages/sharing-smart-contract>
+
+## DataProtectorSharing
+
+Contract module which provides a mechanism for managing and sharing protected
+data through collections, subscriptions, rentals, and sales. This contract
```

Comment:
Make a sentence?

```suggestion
This is a contract module that provides a mechanism for managing and sharing protected
data through collections, subscriptions, rentals, and sales. This contract
```

Can we remove the term "module" or does it help to understand?

```suggestion
This is a contract that provides a mechanism for managing and sharing protected
data through collections, subscriptions, rentals, and sales. This contract
```